src/main/java/core/basesyntax/strategy/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public Fruit performOperation(FruitTransaction fruitTransaction) {
+        return fruitTransaction;
```

Comment:
where is logic?

Reply:
I don't understand the difference between different activities. It seems like balance, supply and return are basically the same thing under a different name - all of them add fruits to the storage. Could you please explain the difference?